### PR TITLE
Add example plans for generic Fedora tests

### PR DESCRIPTION
**Pull request by psss**
@thrix, @bookwar, please have a look.

### CODE REVIEW

File: examples/fedora/rpminspect.fmf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    summary:
+        Generic rpminspect test in a prepared container
+    provision:
+        how: container
+        image: rpminspect-container-name
+    execute:
+        how: shell
+        script: rpminspect -T licence,manpage,disttag *.rpm
+
+/plugin:
```
Comment (thrix) [inline examples/fedora/rpminspect.fmf]: I will remove this as discussed

**Comment by thrix** [comment]
@psss extended after out discussion, added specification for possible future download module

File: examples/fedora/rpminspect.fmf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        # e.g. tmt run -a execute --script "rpminspect BEFORE_NVR AFTER_NVR"
+
+/container-with-download:
+    summary:
+        Generic rpminspect comparison test in a prepared container with profile 'gating'.
+        Uses download prepare step to download the artifacts.
+    provision:
+        how: container
+        image: rpminspect-container-name
+    prepare:
```
Comment (psss) [inline examples/fedora/rpminspect.fmf]: Thinking about the actual usage: Do you expect that specifying a concrete artifact as part of the plan would be a common use case? I would expect this is something that would be almost always provided from the command line. On the other hand it makes sense to keep both L2 keys and command line options consistent.

File: spec/steps/prepare.fmf [hunk 3]
```diff
@@ -40,3 +40,33 @@ description: |
...
+
+        Currently these artifacts are supported:
+            * brew or koji builds - can be specified by URL
+    example: |
+        prepare:
+            how: download
+            artifact: https://koji.fedoraproject.org/koji/buildinfo?buildID=1453726
+            destination: /tmp/systemd-244.1-3.fc32
+
+/download-multi:
```
Comment (psss) [inline spec/steps/prepare.fmf]: I think we can group both examples under a single story. Will be better for tracking and also generated online docs will be more clear.

File: examples/fedora/rpminspect.fmf [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        - how: download
+          artifact: https://koji.fedoraproject.org/koji/buildinfo?buildID=1453726
+          destination: /tmp/systemd-244.1-3.fc32
```
Comment (pvalena) [inline examples/fedora/rpminspect.fmf]: I wonder whether this shouldn't be simply relative path (without the `tmp`), as in prepare we definitely need to [download the builds](https://github.com/psss/tmt/pull/159/files#diff-4476c8abdfef907fe037b9351ba16b5cR158-R159) before install anyway. So could we maybe make the downloaded path symlinked from where the script runs?

**Comment by psss** [comment]
Generic tests are now hosted here: https://github.com/fedora-ci/generic-tests

**Pull request closed without merging**